activity_definitions/configuration.yaml

```diff
@@ -0,0 +1,12 @@
+conduction of simple threat modeling on a technical level:
```

Comment:
A product owner shouldn't be able to change threshold or level.
See also https://github.com/devsecopsmaturitymodel/metricAnalyzer/pull/11/files .

While initial developing, it is ok to do it. Soon it should be changed

Reply:
The configuration.yaml is intended to be set only once by a member of the security team. This was my interpretation of the setup.